src/core_plugins/timelion/public/partials/docs/tutorial.html

```diff
@@ -1,186 +1,380 @@
+          <li><strong>Time range</strong>
+            <p>
+              Use the timepicker <i class="fa fa-clock-o"></i> in the
+              Kibana toolbar to select the time period that contains the
+              data you want to visualize. Make sure you select a time
+              period that includes all or part of the time range shown
+              above.
+            </p>
+          </li>
+        </ul>
+        <p>Now, you should see a line chart that displays a count of your
+          data points over time.
         </p>
       </div>
       <div class="doc-container-buttons">
-        <button ng-click="page = page-1" class="btn btn-primary btn-doc-prev">Previous</button>
-        <button ng-click="page = page+1" class="btn btn-primary btn-doc-next">Next</button>
+        <button
+          ng-click="page = page-1"
+          class="btn btn-primary btn-doc-prev"
+        >
+          Previous
+        </button>
+        <button
+          ng-click="page = page+1"
+          class="btn btn-primary btn-doc-next"
+        >
+          Next
+        </button>
       </div>
     </div>
   </div>
-
   <div ng-show="page === 3">
     <div class="doc-container-content">
-      <h2>Elasticsearch querying in short</h2>
+      <h2>Querying the Elasticsearch datasource</h2>
       <p>
-        We're going to start off talking about the Elasticsearch datasource, because we've already validated that one works for you. Enter <code>.es(*)</code> in the expression input, if its not there already. Hit enter.
+        Now that we've validated that you have a working Elasticsearch
+        datasource, you can start submitting queries. For starters,
+        enter <code>.es(*)</code> in the input bar and hit enter.
       </p>
       <p>
-        This said <em>"hey elasticsearch, find everything in my default index"</em>. If you wanted to find a subset you might do something like <code>.es(html)</code> to count events matching html, or <code>.es('user:bob AND bytes:>100')</code> to find events with <em>bob</em> in the <em>user</em> field, and a bytes field that is greater than 100. Note that we surrounded our query in single quotes this time, because it has spaces. You can enter any lucene query string as the first argument to the <code>.es()</code> function.
-
+        This says <em>hey Elasticsearch, find everything in my default
+        index</em>. If you want to find a subset, you could enter something
+        like <code>.es(html)</code> to count events that match <em>html</em>,
+        or <code>.es('user:bob AND bytes:>100')</code> to find events
+        that contain <em>bob</em> in the <code>user</code> field and have a
+        <code>bytes</code> field that is greater than 100. Note that this query
+        is enclosed in single quotes&mdash;that's because it contains
+        spaces. You can enter any
+        <a href="https://www.elastic.co/guide/en/elasticsearch/reference/5.1/query-dsl-query-string-query.html#query-string-syntax">
```

Comment:
We should probably add `target="_blank"` here to avoid navigating out of Kibana.